src/rnn/legacy/rnnlmlib_static.cpp

```diff
@@ -71,7 +71,7 @@ void CRnnLM_stat::ReadFRHeader(FILE *file) { /*{{{*/
   }
 
   // direct
-  unsigned long int maxent_hash_size;
+	int64_t maxent_hash_size;
```

Comment:
correct tabulation? i now it is old code but still

Reply:
I think its tabs vs spaces (boo on you, Visual Studio), and I couldn't get git-clang-format script working on Mingw (I haven't tried much though).